combat-trainer.lic

```diff
@@ -418,18 +422,23 @@ class LootProcess
     mob_noun = DRRoom.dead_npcs.first
     return true if game_state.construct?(mob_noun)
 
+    echo "  should_perform_ritual? #{should_perform_ritual?(game_state)}" if $debug_mode_ct
     if @last_ritual.nil?
-      if @necro_heal
+      if @necro_heal && !game_state.necro_casting?
         game_state.wounds = check_health
         echo "Severity to Wounds: #{game_state.wounds}" if $debug_mode_ct
         do_necro_ritual(mob_noun, 'consume', game_state) unless game_state.wounds.empty?
       end
+      if @make_zombie && !game_state.necro_casting? && !game_state.cfb_active?
+        echo 'Making zombie' if $debug_mode_ct
+        do_necro_ritual(mob_noun, 'arise', game_state)
```

Comment:
`Arise` will consume the corpse right? If so, should we `return false` here so we don't try and do another ritual and/or continue skinning?

Reply:
Yea good idea, we should probably do that with the consume logic in the above block too